users/password-change.php

```diff
@@ -0,0 +1,104 @@
+<?php
+/** 
+ * Postfix Admin 
+ * 
+ * LICENSE 
+ * This source file is subject to the GPL license that is bundled with  
+ * this package in the file LICENSE.TXT. 
+ * 
+ * Further details on the project are available at http://postfixadmin.sf.net 
+ * 
+ * @version $Id$ 
+ * @license GNU GPL v2 or later. 
+ * 
+ * File: password-change.php
+ * Used by users and admins to change their forgotten login password.
+ * Template File: password-change.tpl
+ *
+ * Template Variables:
+ *
+ * tUsername
+ * tCode
+ *
+ * Form POST \ GET Variables:
+ *
+ * fUsername
+ */
+
+if (preg_match('/\/users\//', $_SERVER['REQUEST_URI'])) {
+  $rel_path = '../';
+  $context = 'users';
+} else {
+  $rel_path = './';
+  $context = 'admin';
+}
+require_once($rel_path . 'common.php');
+
+if ($context == 'admin' && !Config::read('forgotten_admin_password_reset') || $context == 'users' && !Config::read('forgotten_user_password_reset'))
+{
+    header('HTTP/1.0 403 Forbidden');
+    exit(0);
+}
```

Comment:
I'd prefer a die() with a useful message.
Attackers will know what a 403 means anyway, and normal users will have a hard time debugging an empty page.